.readthedocs.yaml

```diff
@@ -0,0 +1,11 @@
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+  commands:
+  - pip install nox
+  - nox --session build_docs -- "${READTHEDOCS_OUTPUT}"/html
```

Comment:
I wonder if there's an easy way to reproduce what RTD injects by default... https://github.com/readthedocs/readthedocs.org/blob/fa54900/readthedocs/doc_builder/backends/mkdocs.py#L131-L279

Reply:
Do you suggest copying the code into _scripts/generate_docs.py_ with amendments?

Reply:
No, I wouldn't care in the context of the initial integration. Maybe, explore later.